pkg/agent/route/route_linux.go

```diff
@@ -722,6 +724,12 @@ func (c *Client) Reconcile(podCIDRs []string, svcIPs map[string]bool) error {
 		if desiredPodCIDRs.Has(route.Dst.String()) {
 			continue
 		}
+		// "on-link" is not identified in IPv6 route entries, the peer IPv6 gateways need to
+		// add into IPv6 routes lonely. So don't delete the routes which are added for the peer
```

Comment:
Maybe the comment needs a little adjustment:
IPv6 doesn't support "on-link" route, routes to the peer IPv6 gateways need to be added seperately. So don't delete such routes.

Reply:
Thanks for the review, updated